assembl/views/discussion/views.py

```diff
@@ -233,6 +234,8 @@ def react_view(request, required_permission=P_READ):
         "sentry_dsn": get('sentry_dsn', ''),
         "bugherd_url": bugherd_url
     }
+    resources_hash = get_resources_hash(theme_name)
+    common_context.update(resources_hash)
```

Comment:
You should put this in `get_default_context`, not here. You need to handle login view too.